BasedOnDiagram.sql

```diff
@@ -0,0 +1,60 @@
+CREATE TABLE "patients" (
+  "id" int,
+  "name" varchar(30),
+  "date_of_birth" date,
+  PRIMARY KEY ("id")
+);
+
+CREATE TABLE "medical_histories" (
+  "id" int,
+  "admitted_at" timestamp,
+  "patient_id" int,
+  "status" varchar(40),
+  PRIMARY KEY ("id"),
+  CONSTRAINT "FK_medical_histories.patient_id"
+    FOREIGN KEY ("patient_id")
+      REFERENCES "patients"("id")
```

Comment:
- [ ] Good job on adding foreign keys, but I dont see indexes for any foreign key. kindly consider adding indexes for all the foreign keys in their respective table. I am requiring this not just here but for all the foreign keys.
for example : CREATE INDEX ON medical_histories (patient_id);
requirement:
Remember to add the FK indexes.